backend/src/index.ts

```diff
@@ -1162,18 +1163,18 @@ app.get('/user/keywords', async (req, res: Response) => {
 
   const userID = sessionFromDB.userId;
 
-  const userKeywords = await prisma.user.findFirst({
-    where: {
-      id: userID,
-    },
-    select: {
-      keywords: {
-        select: { text: true },
+    let userKeywords = await prisma.user.findFirst({
+      where: {
+        id: userID,
       },
-    },
-  });
+      select: {
+        keywords: true,
+      }
+    });
+  
+    if (userKeywords === null) return res.status(200).json([]);

```

Comment:
would you mind changing this to 404? nothing wrong with your approach, but just to keep consistent with how rest of app has panned out